### PR TITLE
Add notes of Egeria workshop October 2019

### PR DESCRIPTION
**Pull request by cong78**
Everyone is welcomed to add more or change things on this workshop note. 

Signed-off-by: Cong Chen <cong.chen@ing.com>

### CODE REVIEW

**Comment by cong78** [comment]
> You need to add the appropriate license - copy from one of our other .md files. RAT check is failing on PR

oops forgot that. I will add it soon

File: developer-resources/workshops/October-2019-Amsterdam-workshop/docs/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+- Join development open source with Linux.
+- JSON-LD scanned into open-metadata-archive.
+   - See area 5 models upcoming.
+   - see book on common information models by Mandy.
+### SAS
+- How SAS is creating metadata modelsOpen Source the SAS mappings
+- Possibility of unify existing SAS metadata model 
+- Prepackage of SAS container for evaluation purpose
+### Magic Box of Security
+- Column based security tag estimation
```
Comment (danielaotelea) [inline developer-resources/workshops/October-2019-Amsterdam-workshop/docs/README.md]: What do you mean with this column based estimation?

File: developer-resources/workshops/October-2019-Amsterdam-workshop/docs/README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+- Location of the asset could change the confidentiality and not linked to asset ownership
```
Comment (danielaotelea) [inline developer-resources/workshops/October-2019-Amsterdam-workshop/docs/README.md]: "Asset" has a relationship with "Location" - "AssetLocation". We agree that we don't use this relationship for the Magic box.

File: developer-resources/workshops/October-2019-Amsterdam-workshop/docs/README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+### Salesforce use case of CIM model
+- Join development open source with Linux.
+- JSON-LD scanned into open-metadata-archive.
+   - See area 5 models upcoming.
+   - see book on common information models by Mandy.
+### SAS
+- How SAS is creating metadata modelsOpen Source the SAS mappings
+- Possibility of unify existing SAS metadata model 
+- Prepackage of SAS container for evaluation purpose
+### Magic Box of Security
```
Comment (danielaotelea) [inline developer-resources/workshops/October-2019-Amsterdam-workshop/docs/README.md]: As a next step that we agreed was to add the "SubjectAria" classification in the algorithm of the Security Officer Connector.

File: developer-resources/workshops/October-2019-Amsterdam-workshop/docs/README.md (hunk 1 above)
Comment (cong78) [reply developer-resources/workshops/October-2019-Amsterdam-workshop/docs/README.md]: Hi I think it was one of the topic you have discussed during the workshop? 
Not sure the details but probably about security tag on schema attributes? (column)

File: developer-resources/workshops/October-2019-Amsterdam-workshop/docs/README.md (hunk 2 above)
Comment (cong78) [reply developer-resources/workshops/October-2019-Amsterdam-workshop/docs/README.md]: Cool I will update this part.

File: developer-resources/workshops/October-2019-Amsterdam-workshop/docs/README.md (hunk 3 above)
Comment (cong78) [reply developer-resources/workshops/October-2019-Amsterdam-workshop/docs/README.md]: Cool I will update this part.

**Pull request merged**